### DISCUSSION TITLE
Welcome to ProjetoLogin Discussions!

### DISCUSSION THREAD

**Discussion by ErickkADR** [ErickkADR/login-page-jujutsu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GuilhermeLisboa3** [comment]
Como você criou essa logo? utilizou algum site?

**Reply by ErickkADR** [reply]
Olá, Guilherme !!! 
Sim, utilizei um site chamado Canva.com que é uma ferramenta gratuita de design gráfico online que você pode usar para criar posts para redes sociais, apresentações, cartazes, vídeos etc.
Espero ter ajudado !!!
